packages/hash/frontend/src/blocks/page/createSuggester/Suggester.tsx

```diff
@@ -68,9 +68,7 @@ export const Suggester = <T,>({
           borderRadius: "6px",
           boxShadow:
             "0px 20px 41px rgba(61, 78, 133, 0.07), 0px 16px 25px rgba(61, 78, 133, 0.0531481), 0px 12px 12px rgba(61, 78, 133, 0.0325), 0px 2px 3.13px rgba(61, 78, 133, 0.02)",
-          borderWidth: `1px`,
-          borderStyle: "solid",
-          borderColor: palette.gray[20],
+          border: `1px solid ${palette.gray[20]}`,
```

Comment:
Optional: revert? (this change was necessary mid-way this PR, but is a noop now)

Reply:
Thanks. I think we can leave it as it is